openwisp_users/tests/utils.py

```diff
@@ -57,7 +50,13 @@ def _create_operator(self, organizations=[], **kwargs):
         )
         opts.update(kwargs)
         operator = User.objects.create_user(**opts)
-        operator.user_permissions.add(*self.get_operator_permissions())
+        groups = Group.objects.filter(name__in=['Administrator', 'Operator'])
+        operator.groups.set(groups)
+        permissions = Permission.objects.filter(
+            Q(codename__endswith='book') | Q(codename__endswith='shelf')
+        )
+        for permission in permissions:
```

Comment:
you remove this for loop and user `operator.user_permissions.add(*permission)`

Reply:
okay.